raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,335 @@
+            - assert: {from: 2, to: 4, total_deposit: 150, balance: 100, state: "opened"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 50, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 40, state: "opened"}
+      - parallel:
+          name: "Check edge cases again"
+          tasks:
+            # try to open a channel with an offline node, but the channel already exists
+            - open_channel: {from: 0, to: 1, total_deposit: 200, expected_http_status: 409}
+            # Try to make a payment to an offline node
+            - transfer: {from: 0, to: 1, amount: 1, expected_http_status: 409, lock_timeout: 30}
+      - serial:
+          name: "Restarting node B"
+          tasks:
+            - start_node: 1
+      - parallel:
+          name: "Checking for the D <-> C channel state"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 101, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 101, state: "opened"}
+              #      - serial:
+              # name: "99 payments from D to C in a direct channel"
+              # repeat: 99
+              # tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 40, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100, balance: 160, state: "opened"}
+      - transfer: {from: 3, to: 2, amount: 99, lock_timeout: 30}
+      - wait: 5
+      - serial:
+          name: "Checking if payments were made successful"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 200, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 2, state: "opened"}
+            # Also A makes a deposit to A <-> B channel
+            - deposit: {from: 0, to: 1, total_deposit: 260}
+            - wait: 10
+      - parallel:
+          name: "Also check if A -> B -> C -> D path has enough capacity for 200 payments of 1 token"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 200, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 2, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 260, balance: 200, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100, balance: 160, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100, balance: 200, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100, balance: 0, state: "opened"}
+      - serial:
+          name: "Closing down channel E <-> C"
+          tasks:
+            - close_channel: {from: 4, to: 2}
+            - wait: 20
+            - assert: {from: 2, to: 4, total_deposit: 150, balance: 100, state: "closed"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 50, state: "closed"}
+              # - serial:
+        # name: "199 payments from A to D in a channel"
```

Comment:
Please remove this part if it's not going to be used

Reply:
Removed